packages/debugger/lib/session/sagas/index.js

```diff
@@ -133,16 +134,20 @@ function* startFullMode() {
 
   yield* trace.addSubmoduleToCount(listenersToActivate.length);
 
-  //begin any full-mode modules that need beginning
-  yield* txlog.begin();
+  if (yield select(session.status.loaded)) {
```

Comment:
what happens if this isn't true here?

Reply:
Saw that it's the same, it just looked to me like a strange guard with no explanation.

Reply:
Hm, I'd ask if it's worth adding a comment, but I'm not sure that belongs in this PR...

Reply:
hm, I don't recall my feelings from last night in depth anymore, so I guess I'll just resolve this for now... if this line continues to confuse me after you move the various things around, I'll ask for the comment then :)